services/accumulo-service/src/test/java/datawave/microservice/accumulo/lookup/LookupServiceAuditDisabledTest.java

```diff
@@ -0,0 +1,330 @@
+            LookupService.Parameter.CQ + "=cq2",
+            LookupService.Parameter.CQ_ENCODING + "=none");
+        //@formatter:on
+        
+        String rowid = "row3";
+        LookupResponse response = doLookup(defaultUserDetails, path(testTableName + "/" + rowid), queryString);
+        assertEquals("Lookup should have matched 1 entry", 1, response.getEntries().size());
+    }
+    
+    @Test
+    public void testLookupWithBase64Params() throws Exception {
+        
+        //@formatter:off
+        String queryString = TestHelper.queryString(
+            "useAuthorizations=B",
+            "columnVisibility=foo",
+            LookupService.Parameter.CF + "=" + Base64.encodeBase64URLSafeString("cf2".getBytes()),
+            LookupService.Parameter.CF_ENCODING + "=base64",
+            LookupService.Parameter.CQ + "=" + Base64.encodeBase64URLSafeString("cq2".getBytes()),
+            LookupService.Parameter.CQ_ENCODING + "=base64",
+            LookupService.Parameter.ROW_ENCODING + "=base64");
+        //@formatter:on
+        
+        String rowidBase64 = Base64.encodeBase64URLSafeString("row3".getBytes());
+        LookupResponse response = doLookup(defaultUserDetails, path(testTableName + "/" + rowidBase64), queryString);
+        assertEquals("Lookup should have matched 1 entry", 1, response.getEntries().size());
+    }
+    
+    @Test
+    public void testLookupBeginEndSubset() throws Exception {
+        
+        //@formatter:off
+        String queryString = TestHelper.queryString(
+            "useAuthorizations=A,B,C,D,E,F,G,H,I",
+            "columnVisibility=foo",
+            LookupService.Parameter.CF + "=cf2",
+            LookupService.Parameter.BEGIN_ENTRY + "=2",
+            LookupService.Parameter.END_ENTRY + "=5");
+        //@formatter:on
+        
+        String rowid = "row1";
+        LookupResponse response = doLookup(defaultUserDetails, path(testTableName + "/" + rowid), queryString);
+        assertEquals("Lookup should have returned 4 entries", 4, response.getEntries().size());
+        assertTrue("First result should be cq3", ((DefaultKey) (response.getEntries().get(0).getKey())).getColQual().equals("cq3"));
+        assertTrue("Last result should be cq6", ((DefaultKey) (response.getEntries().get(3).getKey())).getColQual().equals("cq6"));
+    }
+    
+    @Test
+    public void testErrorOnBeginGreaterThanEnd() throws Exception {
+        
+        //@formatter:off
+        String queryString = TestHelper.queryString(
+            "useAuthorizations=A,B,C,D,E,F,G,H,I",
+            "columnVisibility=foo",
+            LookupService.Parameter.CF + "=cf2",
+            LookupService.Parameter.BEGIN_ENTRY + "=7",
+            LookupService.Parameter.END_ENTRY + "=5");
+        //@formatter:on
+        
+        try {
+            doLookup(defaultUserDetails, path(testTableName + "/row2"), queryString);
+            fail("This code should never be reached");
```

Comment:
FYI, you can use an ExpectedException @Rule to cover this case. See DatawaveServerPropertiesTest for an example.

Reply:
Fixed in latest commit